content/efcore/getting-started/adding-ef-core.md

```diff
@@ -0,0 +1,99 @@
+| SQLite | [Microsoft.EntityFrameworkCore.SQLite](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.SQLite/) |
+| Npgsql (PostgreSQL) | [Npgsql.EntityFrameworkCore.PostgreSQL](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL) |
+| MySQL (Official) | [MySql.Data.EntityFrameworkCore -Pre](https://www.nuget.org/packages/MySql.Data.EntityFrameworkCore) |
+| Pomelo (MySQL) | [Pomelo.EntityFrameworkCore.MySQL](https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySQL) |
+| Microsoft SQL Server Compact Edition | [EntityFrameworkCore.SqlServerCompact40](https://www.nuget.org/packages/EntityFrameworkCore.SqlServerCompact40) OR [EntityFrameworkCore.SqlServerCompact35](https://www.nuget.org/packages/EntityFrameworkCore.SqlServerCompact35) |
+| IBM Data Servers | [EntityFramework.IBMDataServer -Pre](https://www.nuget.org/packages/EntityFramework.IBMDataServer) |
+| InMemory (for Testing) | [Microsoft.EntityFrameworkCore.InMemory](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.InMemory/) |
+| Devart (MySQL, Oracle, PostgreSQL, SQLite, DB2, SQL Server, and more) | N/A, for installation instructions, follow the [Devart dotConnect documentation](https://www.devart.com/dotconnect/). |
+
+##Create an EF Core Project
+
+To create a new .NET Core project using command line: 
+
+1. Create a new directory for the project.  
+
+	mkdir MyProject 
+	
+2. Navigate to this project directory. 
+
+	cd MyProject 
+	
+3. Initialize a .NET Core project (this creates project.json and Startup.cs) 
+
+	dotnet new 
+
+4. Restore dependencies. 
+
+	dotnet restore 
+
+5. Run the project 
+
+	dotnet run
+
+You can verify that Entity Framework Core is installed by running
+
+    dotnet ef -help
+	
+##Using a Database
+###Code-first vs. Database-first
+
+Now that you have a working .NET Core project, we can connect the code project to a database. EF Core allows developers to generate a database from C# code, or to connect an existing database to the project. These methods are known as "code-first" and "database-first." Code-first projects create a database from models made in-code that represent database tables. Developers can populate tables, delete fields, and define keys all from the code without writing a single SQL script. While developers can still modify tables and data from code, database-first projects connect a EF Core project to an existing database. This works well for developers who already have a database in development. 
+
+In this tutorial, we will demonstrate how to generate a database with a code-first project and how to connect a database for database-first. 
+
+###Code-first: Generating a Database
+
+For both code-first and database-first projects, we must set up the database context. Contexts "point" EF Core to the specified database (*.db) file. DbContext is an existing .NET Core class that can be extended to your context. In this example, we are generating a SQLite database from our code. 
```

Comment:
We need to define what a context is.

Examples from google searches:
* The context class manages the entity objects during run time, which includes populating objects with data from a database, change tracking, and persisting data to the database ([source](https://msdn.microsoft.com/en-us/library/jj729737(v=vs.113).aspx))
* A DbContext instance represents a combination of the Unit Of Work and Repository patterns such that it can be used to query from a database and group together changes that will then be written back to the store as a unit. DbContext is conceptually similar to ObjectContext. ([source](https://msdn.microsoft.com/en-us/library/system.data.entity.dbcontext(v=vs.113).aspx))